### ISSUE TITLE
Release 0.0.5

### ISSUE THREAD

**Issue by ajlanghorn** [goldyfruit/puppet-grub2#9]
Hi @goldyfruit,

Please can you consider releasing 0.5.0? When using this with `librarian-puppet` after my recent changes, the 0.4.0 release is downloaded and cached locally, which omits the feature I just added. The only way to get this working with the 0.4.0 release would be to pin this to a SHA-1 locally, which I'd rather not do if you'd be prepared to release 0.5.0.

Thanks!

Andrew

**Comment by goldyfruit** [comment]
@ajlanghorn Thanks again for the contribution, it's appreciate !
I will release a new version this evening or tomorrow.

**Comment by ajlanghorn** [comment]
@goldyfruit Great, thanks! And, no problem. I liked this module to fix my GRUB2 story internally without having to start using Augeas to manage my configuration files.

**Comment by goldyfruit** [comment]
@ajlanghorn That's why i created this module, Augeas can be too huge for some usage.
I just released the 0.0.5 version \o/

```
# puppet module upgrade goldyfruit-grub2
Notice: Preparing to upgrade 'goldyfruit-grub2' ...
Notice: Found 'goldyfruit-grub2' (v0.0.4) in /srv/data/puppet/envs/production/modules ...
Notice: Downloading from https://forge.puppetlabs.com ...
Notice: Upgrading -- do not interrupt ...
/srv/data/puppet/envs/production/modules
└─┬ goldyfruit-grub2 (v0.0.4 -> v0.0.5)
  └── puppetlabs-stdlib (v4.5.1 -> v4.6.0)
```

**Comment by goldyfruit** [comment]
@ajlanghorn Released 0.0.6 due to this issue #13 
```
# puppet module upgrade goldyfruit-grub2
Notice: Preparing to upgrade 'goldyfruit-grub2' ...
Notice: Found 'goldyfruit-grub2' (v0.0.5) in /srv/data/puppet/envs/production/modules ...
Notice: Downloading from https://forge.puppetlabs.com ...
Notice: Upgrading -- do not interrupt ...
/srv/data/puppet/envs/production/modules
└── goldyfruit-grub2 (v0.0.5 -> v0.0.6)
```

**Comment by ajlanghorn** [comment]
@goldyfruit Thanks for the heads-up!
